### PR TITLE
Updated CONFIGURATION section to capture settings options

### PR DESCRIPTION
**Pull request by validbeck**
## Internal Notes for Reviewers

> For sc-2431, I made some changes to the `CONFIGURATION` section under **Guides**.

### Manage your organization

| Onboarding Guides | Manage your organization |
|---|---|
| Screenshot 2024-07-02 at 4 30 19 PM | Screenshot 2024-07-03 at 10 37 55 AM |

> Net-new page — includes switching between organizations and changing organization names — that wraps the following two pages under it:
>
> 1. Set up your organization
> 2. Configure AWS PrivateLink

#### Set up your organization

| Old | New |
|---|---|
| Screenshot 2024-07-02 at 4 33 37 PM |  Screenshot 2024-07-02 at 4 34 09 PM |

>  I edited this for style guide adherence and accuracy as it was a bit rough beforehand. 
- I made sure that the training that references it looks OK too. 

| Old | New |
|---|---|
| Screenshot 2024-07-02 at 4 40 26 PM | Screenshot 2024-07-02 at 4 36 27 PM |

### Settings icons
I also made sure any references to **Settings** had the fa-gear added in front of it. :) 

### Changed files
**NB:** Again, I tried to only include the edited files for review, but the merge from main changed a bunch of stuff...

### CODE REVIEW

File: site/_quarto.yml [hunk 1]
```diff
@@ -161,15 +161,17 @@ website:
         - guide/guides.qmd
         - text: "---"
         - text: "CONFIGURATION"
-        - guide/configuration/set-up-your-organization.qmd
+        - file: guide/configuration/manage-your-organization.qmd
```
Comment (nrichers) [inline site/_quarto.yml:164]: Given that this topic has child topics under it, the title should be "Managing your organization" and the file name should match that. 

(FWIW, "Onboarding users" likely should also be "Managing users" as we also cover removing users.)

Comment (validbeck) [reply site/_quarto.yml:164]: I will change both of those!

File: README.md [hunk 2]
```diff
@@ -25,22 +25,21 @@ You need:
 - For Windows operating systems, install the `make` command via [Cygwin](https://cygwin.com/install.html)
 
 ## How to contribute
-> First, read through and familiarize yourself with our [ValidMind style guide](https://docs.validmind.ai/about/style-guide.html).
 
-Our core docs are sourced in Quarto Markdown under [`site`](https://github.com/validmind/documentation/tree/main/site/). If you create new documentation, make sure to add it to the [`_quarto.yml`](https://github.com/validmind/documentation/blob/main/site/_quarto.yml) file.
+Our core docs are sourced in Quarto Markdown under [site/guide/](https://github.com/validmind/documentation/tree/main/site/guide). If you create new documentation, make sure to add it to the [_quarto.yml](https://github.com/validmind/documentation/blob/main/site/_quarto.yml) file.
```
[the remark this answers is not in the document]
Comment (validbeck) [reply README.md:29]: Hm, I had kind of did this deliberately — most of our "core" stuff to me is the how-to, not any of the quickstart, conceptual or developer framework references.

File: README.md [hunk 3]
```diff
@@ -2,13 +2,13 @@
...
 
-This is the home for the user-facing documentation and related infrastructure for ValidMind. If you want to make updates to our external docs site, you're in the right place!
+This is the home for the user-facing documentation and related infrastructure for ValidMind. If you want to make updates to our external docs site, you're in the right place.
 
 ## Contributing to the documentation
 
 We believe in the power of collaboration and welcome contributions from the community. If you've noticed an issue or have ideas for improvement, please create a pull request or submit an issue. A member of ValidMind's documentation team will review your suggestions and support you in contributing to the docs. 
 
-- Follow the [Quickstart](https://docs.validmind.ai/guide/quickstart-try-developer-framework-with-jupyterhub.html) for a 15-minute introduction to the Developer Framework and ValidMind Platform.
+- Follow the [Quickstart](https://docs.validmind.ai/guide/quickstart.html) for a 20 minute introduction to the Developer Framework and ValidMind Platform.
```
[the remark this answers is not in the document]
Comment (validbeck) [reply README.md:11]: These changes were made in another PR and just haven't been fully merged yet, my bad.

**Comment by validbeck** [comment]
> I'm confused by this PR: besides the content changes to our user guides, it seems to UNDO several recent changes to our README.md, for example. Does your PR branch need to be updated before I continue reviewing?

Yes, I didn't add all of the files merged from main is why it looks like this, I was trying to keep it to the files I changed but borked it.

**Comment by validbeck** [comment]
@nrichers Changes:

| README* | Manage your organization | Onboarding users |
|---|---|---|
| Screenshot 2024-07-04 at 10 09 28 AM |  Screenshot 2024-07-04 at 10 06 17 AM |Screenshot 2024-07-04 at 10 06 25 AM |

> *FTR I've already updated the broken links in [this working branch](https://github.com/validmind/documentation/tree/beck/sc-5132/fix-broken-external-links-out-to-restructured) as part of the broken links Story, so that's taken care of!

File: README.md [hunk 4]
```diff
@@ -27,7 +27,8 @@ You need:
 ## How to contribute
 > First, read through and familiarize yourself with our [ValidMind style guide](https://docs.validmind.ai/about/style-guide.html).
 
-Our core docs are sourced in Quarto Markdown under [`site`](https://github.com/validmind/documentation/tree/main/site/). If you create new documentation, make sure to add it to the [`_quarto.yml`](https://github.com/validmind/documentation/blob/main/site/_quarto.yml) file.
+- Our core docs are sourced in Quarto Markdown under [`site/guide`](https://github.com/validmind/documentation/tree/main/site/guide).
```
Comment (nrichers) [inline README.md:30]:
```suggestion
- Our core user guides are sourced in Quarto Markdown under [`site/guide`](https://github.com/validmind/documentation/tree/main/site/guide). 
```

I know you mentioned that this path choice is deliberate but it's still confusing to me, as there are lots of other things I would consider 'core docs' that aren't in this path. For example, the getting-started topics to me qualify as core docs, as they enable you to start using our product. Can we settle on this compromise?

**Review by nrichers — APPROVED** [review]
Thanks for sorting out the merge issues with this branch — changes LGTM now. Just one semantic quibble over what we mean by 'core docs', see my suggestion. :shipit:

**Pull request merged**